docs/Settings.md

```diff
@@ -4362,6 +4362,16 @@ Number of decimals for the battery voltages displayed in the OSD [1-2].
 
 ---
 
+### osd_msp_displayport_fullframe_interval
+
+P Frame interval for MSP DisplayPort [seconds] 0 = disabled
```

Comment:
@MrD-RC as I was thinking about that as well, I think we should approach this problem in a slightly different way. 
* `-1` - disabled
* `0` - every frame
* `>0` every so many seconds

And I'm also leaning towards having this enabled by default with something like `5s` interval

Reply:
Cool. I'll make it so.